fix: Linux CI fails due to missing freetype2 dev headers

## Problem

Linux CI jobs (Check, Clippy, Test) are failing with `freetype2` package not found:

```
configure: error: Package requirements (freetype2) were not met:
Package 'freetype2', required by 'virtual:world', not found
```

This is caused by `servo-fontconfig-sys` build script requiring `freetype2` dev headers, which are not installed in the CI runner environment.

## Affected Jobs

- Check (`cargo check --all-features`)
- Clippy (`cargo clippy --all-features -- -D warnings`)
- Test (`cargo test --all-features`)

Windows Check passes since it doesn't depend on fontconfig/freetype2.

## Observed In

- PR #291 run: https://github.com/ugai/sldshow2/actions/runs/22625913825

## Proposed Fix

Add `libfreetype6-dev` (and possibly `libfontconfig1-dev`) to the CI runner's apt install step, or pin the dependency to avoid the system freetype requirement.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -20,7 +20,7 @@ jobs:
       - name: Install Linux dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev
+          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev libfontconfig1-dev
       - uses: Swatinem/rust-cache@v2
       - run: cargo check --all-features
 
@@ -45,7 +45,7 @@ jobs:
       - name: Install Linux dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev
+          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev libfontconfig1-dev
       - uses: Swatinem/rust-cache@v2
       - run: cargo clippy --all-features -- -D warnings
 
@@ -58,7 +58,7 @@ jobs:
       - name: Install Linux dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev
+          sudo apt-get install -y libasound2-dev libudev-dev libxkbcommon-dev libwayland-dev libfontconfig1-dev
       - uses: Swatinem/rust-cache@v2
       - run: cargo test --all-features
 

